x/staking/legacy/v0_36/types.go

```diff
@@ -41,6 +41,20 @@ type (
 		MinSelfDelegation       sdk.Int                 `json:"min_self_delegation" yaml:"min_self_delegation"`
 	}
 
+	bechValidator struct {
```

Comment:
@kwunyeung can we also do the same change for 0.37 in staking?

Reply:
Ah... I see there is a `security contact` in `description` in v0.37.

Reply:
@alexanderbez updated for 0.37 as well.